### ISSUE TITLE
UI / CSS library to build the UI of the plugin

### ISSUE THREAD

**Issue by phil-lgr** [oriziv/figma-sass-less-plugin#4]
What about https://github.com/thomas-lowry/figma-plugin-ds?

I think this is what you put in figma-plugin-ui-css-0.5 right?

**Comment by phil-lgr** [comment]
related: https://github.com/thomas-lowry/figma-plugin-ds/pull/11

**Comment by oriziv** [comment]
Yes its the same thing. i took if from Figma docs, we can use it
